keps/NNNN-kep-template/README.md

```diff
@@ -473,6 +473,21 @@ checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
 
+###### How can an end user leveraging this feature know that it is working?
+
+<!--
+Pick one more of these and delete the rest.
+Recall that end users cannot usually observe component logs or access metrics.
```

Comment:
maybe something like: "Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement and operation of this feature. Recall that end users cannot..."

but also wondering do we also want this mandatory in docs (if it's not already)?